tools/bin/mbedtls-archive-epic.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+""" Assist with the transition of a projectv1 to a projectv2. 
```

Comment:
Or between projectv2, surely? AFAICT this does work FROM v2 as well, and it's what we want going forward.